packages/react/test/specs/commonTests/isConformant.tsx

```diff
@@ -53,40 +58,40 @@ export default (Component, options: Conformant = {}) => {
 
   const componentType = typeof Component
 
-  // This is added because the component is mounted
+  const helperComponentNames = [
+    ...[ThemeProvider, FelaTheme, Ref, RefFindNode],
+    ...(usesWrapperSlot && [Box]),
+  ].map(getDisplayName)
+
+  const toNextNonTrivialChild = (from: ReactWrapper) => {
```

Comment:
It will be great to extract this code outside the `isConformant()` function.

Reply:
would rather like to leave this logic here, so that all the bits related to traversal logic would be kept at the same module. Is there any place where we need this logic as well now?

Reply:
Or may be the whole `getComponent()`... To concentrate `isConformant()` on assertions

Reply:
would rather like to wait with segregating this logic, as cannot see any other place where it might be useful now. Yes, agree with the fact that it will make `isConformant` module to be focused on assertions, but, at the same time, it will introduce a module with the functionality that currently suits and specific for the only client (`isConformant`). 

What if we would wait with this split a bit when there will be more appealing reason? Another point is that we are not making things any worse with that, as this logic was always part of conformance tests, and there were no other module so far interested in this functionality